multi_input_tests.py

```diff
@@ -0,0 +1,87 @@
+import argparse
```

Comment:
this file should be removed at the end no?

Also, I recommend you to use `pytest -k keyword` to run only part of the test.

Reply:
I can convert this file to be a test and place it with the rest of the tests and label it `pytest.mark.slow` as suggested earlier. Would that be a good idea going forward?

Reply:
yes!